src/models/gg-model-base.cc

```diff
@@ -0,0 +1,56 @@
+#include "gg-model-base.hh"
+#include "thunk_writer.hh"
+
+#include <cstdlib>
+#include <iostream>
+#include <fstream>
+#include <sys/types.h>
+#include <sys/stat.h>
+
+using namespace std;
+using namespace gg::thunk;
+
+const char *GGModelBase::GG_DIR_FLAG = "GG_DIR";
```

Comment:
Let's just have this as an `std::string`, and use `string::c_str()` whenever it's necessary to pass a `char *`.